src/Silex/CallbackResolver.php

```diff
@@ -33,35 +33,64 @@ public function __construct(Container $app)
      */
     public function isValid($name)
     {
-        return is_string($name) && preg_match(static::SERVICE_PATTERN, $name);
+        return is_string($name) && (
+            preg_match(static::SERVICE_PATTERN, $name) ||
+            isset($this->app[$name])
+        );
+    }
+
+    /**
+     * Throws an InvalidArgumentException for the service not existing.
+     *
+     * @param string $name
```

Comment:
$service, not $name but anyway, the whole docblock should be removed as we don't use them for private methods (except when it is really needed, which is not the case here).